airbyte-cdk/python/unit_tests/sources/utils/test_schema_helpers.py

```diff
@@ -62,7 +92,10 @@ def test_inline_schema_resolves():
             "properties": {
                 "str": {"type": "string"},
                 "int": {"type": "integer"},
-                "obj": {"type": ["null", "object"], "properties": {"k1": {"type": "string"}}},
+                "obj": {
```

Comment:
please use pre-commit command for code formatting. It would eliminate those autogenerated changesets.
```bash
pip install pre-commit
cd <project root>
pre-commit install
```

Reply:
done